pkg/kritis/cryptolib/pgp_test.go

```diff
@@ -20,11 +20,14 @@ import (
 	"testing"
 )
 
-// These keys and signatures generated by the following commands:
+// These keys and signatures were generated by the following commands:
 // `gpg --quick-generate-key --yes pgp@cryptolib.com`
 // `gpg --export --armor pgp@cryptolib.com`
 // `gpg --output gpgSignature --armor -u pgp@cryptolib.com --sign payload`
 
+const payload = `this is the payload i wish to sign
```

Comment:
payload-> testPayload

nit: **T**his is the payload **I** wish to sign, and remove the trailing newline

Reply:
Done.